src/main/java/entities/computers/ModelOneLaptop.java

```diff
@@ -0,0 +1,58 @@
+package entities.computers;
+
+import interfaces.Computer;
+import interfaces.Monitor;
+import interfaces.Processor;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.extern.log4j.Log4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+
+@Getter
+@Setter
+@Log4j
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_SINGLETON)
+@PropertySource({"classpath:OneLaptop.properties"})
+public class ModelOneLaptop implements Computer {
+
+    protected Processor processor;
+    protected Monitor monitor;
+
+    @Value("${size}")
```

Comment:
Value-form property file sets incorrectly. In file it is 70, but in log I see 100. If  I change annotation to this -@Value("${unknown.param:70}") 
it works and sets 70, but it is not correct imao, 
can you explain what I'm doing wrong (why 70 isn't set properly)?

Reply:
It should be 70. Please provide your log file.

Reply:
The hotfix is to use different names for the key.
size1=100
size2=75
But you should google how to fix it more elegant.